artikel/2015-07-31-Hitzewelle-Sommerspaß-oder-Klimawandel.md

```diff
@@ -0,0 +1,34 @@
+#Hitzewelle: Sommerspaß oder Klimawandel?
+
+http://www.dw.com/de/hitzewelle-sommerspa%C3%9F-oder-klimawandel/a-18616694
+
+der Artikel "Hitzewelle: Sommerspaß oder Klimawandel?" spricht ein
+interessantes und wichtiges Thema an, weil wir diesen Sommer so viel 
+Hitzwelle in Deutschland gehabt haben.
+
+Der Artikel geht um Klimawandel und die Möglichkeit, ob es existiert. In 
+2015 gab es rekorde Temperaturen und häufiger Hitzwelle in Deutschland. 
+Im Juli 2015 gab es einen neuen Hitzrekord in Bayern mit 40,3 Grad Celsius 
+gemessen. Obwohl das heiße Wetter schön für Deutschen ist, finden die 
+meisten, dass ihre Wohnungen nicht für das heiße Wetter gut gebaut sind.
+
+Einerseits freuen die meisten Deutschen auf das heiße Wetter, andereseits 
+haben manche Deutschen Angst vor dem Klimawandel, denn sie glauben, dass 
+Menschen den Planeten erwärmen. Eine Expertin, die Dim Coumou heißt, 
+vom Potsdam-Institut für Klimafolgenforschung ist sicher, dass die Hitze 
+mit dem Klimawandel zu tun hat, weil Deutschland normalerweise nicht so 
+häufiger Hitzwelle hat.
+
+Andere Menschen wie Florian Imbery sagen, dass Klimawandel keine 
+Rolle für diese Hitzwelle spielt. Er arbeitet beim Deutschen Wetterdienst 
+und er glaubt, dass wir nicht so schnell die Schuld an Hitzwelle auf 
+Klimawandel schieben. Er denkt, dass wir viel mehr Datei sammeln müssen 
```

Comment:
Hmm, I would close this with `... die Schuld an der Hitzewelle auf den Klimawandel schieben sollten.`